### DISCUSSION TITLE
MSI Z390M  MORTAR用哪个EFI呀，可以装黑苹果不

### DISCUSSION THREAD

**Discussion by LWQCYH** [daliansky/Hackintosh#255] [General]
MSI Z390M  MORTAR哪个用哪个EFI呀，可以装黑苹果不
配置如下
CPU：8700
主板：MSI Z390M MORTAR
显卡：华硕RX580 2304SP
硬盘：西数SN750 1T+三星PM961 250G

**Comment by daliansky** [comment]
https://github.com/kimmai0322/MSI-Z390M-Mortar-opencore
